subprojects/file-watching/src/integTest/groovy/org/gradle/internal/watch/VirtualFileSystemRetentionIntegrationTest.groovy

```diff
@@ -723,6 +724,29 @@ class VirtualFileSystemRetentionIntegrationTest extends AbstractIntegrationSpec
         buildDir << ["build", "build/myProject"]
     }
 
+    def "can remove watched directory after all files inside have been removed"() {
```

Comment:
I think some explanation here would be good about this being targeted at Windows.

Reply:
We should also link to the issue here.

Reply:
Done.